docs/release-notes.rst

```diff
@@ -6,8 +6,35 @@ This document describes major installer-related changes in Fedora releases.
 Fedora 38
 #########
 
-Architecture support changes
-----------------------------
+Changes in the graphical interface
```

Comment:
I have checked my pull requests for F38 and https://github.com/rhinstaller/anaconda/pull/4501 might be documented as well. It is a fix of a bug, that was ignored for a very long time,  and it might be interesting for current RHEL branches.

Reply:
Thank you, added.